Module/templates/bug-report.html

```diff
@@ -112,6 +111,9 @@ <h4 class="flexrow">
     <div class="form-group">
       <label for="sendActiveModules">{{localize 'BUG.form.options.activemod'}}</label>
       <input type="checkbox" id="sendActiveModules" name="formFields.sendActiveModules" checked />
+
```

Comment:
this should be split into two form-group s
```suggestion
</div>
<div class="form-group">
```